packages/react-icons/src/utils/createSvgIcon.ts

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+// @ts-ignore
+import { css, getNativeProps, htmlElementProperties } from '@uifabric/utilities/src';
```

Comment:
Please don't use ts-ignore except as a last resort. Is there some reason this didn't work?
```suggestion
import { css, getNativeProps, htmlElementProperties } from '@uifabric/utilities';
```

Reply:
Sry very early prototype, not a prod ready version, just was looking for an early review on API and general stuff..